src/views/Validator/ValidatorContent.tsx

```diff
@@ -106,7 +109,11 @@ export function ValidatorContent({
                     flex: 1;
                   `}
                 >
-                  <ReadOnlyEditor language="javascript" value={script} />
+                  <ReadOnlyEditor
+                    language="javascript"
+                    value={script}
+                    onCopy={handleCopy}
+                  />
```

Comment:
Nit: `ReadOnlyEditor` should probably also be used in Script Preview, but it's out of scope of this PR.